CSharpSourceCode/Abilities/Ability.cs

```diff
@@ -185,48 +224,42 @@ private void AddLight(ref GameEntity entity)
 
         private void AddPhysics(ref GameEntity entity)
         {
-            if (!IsStaticAbility()) 
```

Comment:
you do the same thing for both cases?

Reply:
I think we should check if the ability is dynamic before run this method

Reply:
but you do the same thing irrespective what you choose... or did I read wrong

Reply:
yes, it's the same thing, but i think it looks more logically